Dockerfile

```diff
@@ -4,10 +4,9 @@ ENV CGO_ENABLED=0
 
 WORKDIR /build
 COPY . /build
-
 # SINKER_VERSION is set during the release process
 ARG SINKER_VERSION=0.0.0
-RUN go build -ldflags="-X 'github.com/plexsystems/sinker/internal/commands.sinkerVersion=${SINKER_VERSION}'"
+RUN go build -tags 'containers_image_openpgp' -ldflags="-X 'github.com/plexsystems/sinker/internal/commands.sinkerVersion=${SINKER_VERSION}'"
```

Comment:
@jpreese fyi, this will affect the build pipeline too.
As per https://github.com/containers/image/blob/main/README.md#supported-build-tags
```
Use a Golang-only OpenPGP implementation for signature verification instead of the default cgo/gpgme-based implementation; the primary downside is that creating new signatures with the Golang-only implementation is not supported.
```
Which should not affect us, as I don't think generating signatures would be outside the problem domain sinker addresses.